observatory-platform/observatory/platform/terraform/main.tf

```diff
@@ -219,15 +218,63 @@ resource "google_storage_bucket_iam_member" "observatory_transform_bucket_observ
   member = "serviceAccount:${google_service_account.observatory_service_account.email}"
 }
 
-resource "google_storage_bucket_iam_member" "observatory_transform_bucket_observatory_service_account_object_creator" {
+# Must have object admin so that files can be overwritten
+resource "google_storage_bucket_iam_member" "observatory_transform_bucket_observatory_service_account_object_admin" {
   bucket = google_storage_bucket.observatory_transform_bucket.name
-  role = "roles/storage.objectCreator"
```

Comment:
as discussed, perhaps wrap up some of the permissions changes into their own pull request